### ISSUE TITLE
Are models downloadable as zip archives from zenodo?

### ISSUE THREAD

**Issue by k-dominik** [bioimage-io/spec-bioimage-io#121]
Hello @oeway,

quick question: I wanted to try one of the modelzoo compliant networks @constantinpape has trained, brought up to spec and uploaded it (via the bioimageio uploader as it says on zenodo): [model](https://sandbox.zenodo.org/record/865618#.YMnEoCWxVpQ).

I'm not sure I've overlooked it, but it seems that all files have to be downloaded separately? Or is the a secret download all button somewhere?

Cheers

**Comment by oeway** [comment]
You are right on spot, Tom also raised the same question: https://github.com/bioimage-io/bioimage.io/pull/104#issuecomment-859363275

The idea is to use the bioimage.io as the user interface and the user will click the packager icon to download. As a fallback, we will add a link to the zenodo page to bring the user back to the packager for downloading.

I need to fix the packager to fit the recent changes. You will be able to download all by then.

**Comment by constantinpape** [comment]
> The idea is to use the bioimage.io as the user interface and the user will click the packager icon to download.

How close are we to having this (as well as the update button) on the main website? I think it would be important to have this for the course that @akreshuk @esgomezm @k-dominik are running end of June.

**Comment by oeway** [comment]
> > The idea is to use the bioimage.io as the user interface and the user will click the packager icon to download.
> 
> How close are we to having this (as well as the update button) on the main website? I think it would be important to have this for the course that @akreshuk @esgomezm @k-dominik are running end of June.

If it's only the packager, 30min to 1h of work needed. Do you see other major problem with the preview version of the upload feature?

Another questions, would it work if we stick with the sandbox.zenodo.org for the course? Since we cannot easily take down deposition on zenodo, we should not switch to the production server until we are fully ready.

**Comment by constantinpape** [comment]
> If it's only the packager, 30min to 1h of work needed.

Awesome!

> Do you see other major problem with the preview version of the upload feature?

No, I think we should go ahead and merge it.
There are still the two issues I have with firefox on my laptop and chromium, but I can just make separate issues for this once the PR is merged.
The upload button already makes it sooooooooooooooo much more convenient to publish models.

> Another questions, would it work if we stick with the sandbox.zenodo.org for the course? Since we cannot easily take down deposition on zenodo, we should not switch to the production server until we are fully ready.

I think it's fine to stay on zenodo.sandbox for the course.

**Comment by k-dominik** [comment]
thx for the replies @oeway and @constantinpape. And of course for the solution, too :). `sandbox.zenodo.org` is fine!

**Comment by oeway** [comment]
FYI: I just fixed the model packager, and new uploaded models will contain a download package link in the description on zenodo. The download link is not implemented yet, but it's good to already insert links.


BTW, I will move this issue to bioimage.io repo since it's not related to the spec.

We have also this issue to keep track of the download all feature: https://github.com/bioimage-io/bioimage.io/issues/108
